presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -187,3 +187,46 @@ Accessors
 .. function:: ST_NumInteriorRing(Geometry) -> bigint
 
     Returns the cardinality of the collection of interior rings of a polygon.
+
+=====================
+Bing Tiles
+=====================
+
+A set of functions to convert between geometries and
```

Comment:
Why is there a new line after `and`?

Reply:
@electrum asked me to keep lines within 80-100 chars long. Since the next thing after 'and' is a long URL, I had to put a line break here.

Reply:
With the new line when you build `presto-docs` the output looks weird.
![screen shot 2017-10-04 at 5 33 32 pm](https://user-images.githubusercontent.com/1223839/31206013-3d1fa4a6-a92a-11e7-9a55-4a69e6462777.png)